src/components/Message/Message.test.tsx

```diff
@@ -8,174 +8,15 @@ describe('Message component', () => {
-                }
-              }
-            >
-              <View
-                style={
-                  Object {
-                    "alignSelf": "flex-start",
-                  }
-                }
-              >
-                <Text
-                  color="main"
-                  fontSize={16}
-                  fontWeight="medium"
-                  horizontalTextAlign="left"
-                  style={
-                    Array [
-                      Object {
-                        "color": "#000000",
-                        "fontFamily": "Rubik-Medium",
-                        "fontSize": 16,
-                        "textAlign": "left",
-                        "textAlignVertical": "center",
-                      },
-                    ]
-                  }
-                  verticalTextAlign="center"
-                >
-                  holmes
-                </Text>
-              </View>
-              <View
-                style={
-                  Object {
-                    "alignSelf": "flex-start",
-                    "paddingLeft": 8,
-                    "paddingTop": 2,
-                  }
-                }
-              >
-                <Text
-                  color="subtitle"
-                  fontSize={14}
-                  horizontalTextAlign="left"
-                  style={
-                    Array [
-                      Object {
-                        "color": "#999999",
-                        "fontFamily": "Rubik-Regular",
-                        "fontSize": 14,
-                        "textAlign": "left",
-                        "textAlignVertical": "center",
-                      },
-                    ]
-                  }
-                  verticalTextAlign="center"
-                >
-                  1:30pm
-                </Text>
-              </View>
-            </View>
-            <View
-              style={
-                Object {
-                  "flexShrink": 1,
-                }
-              }
-            >
-              <Text
-                color="main"
-                fontSize={14}
-                horizontalTextAlign="left"
-                style={
-                  Array [
-                    Object {
-                      "color": "#000000",
-                      "fontFamily": "Rubik-Regular",
-                      "fontSize": 14,
-                      "textAlign": "left",
-                      "textAlignVertical": "center",
-                    },
-                  ]
-                }
-                verticalTextAlign="center"
-              >
-                Brownie powder marshmallow dessert carrot cake marzipan cake caramels. Muffin topping wafer jelly apple pie candy. Fruitcake chocolate pudding fruitcake candy lemon drops chocolate.
-              </Text>
-            </View>
-          </View>
-        </View>
-      </View>
-    `);
+    expect(toJSON()).toMatchInlineSnapshot();
```

Comment:
that's a bid odd that this tests passes. I think it means that the component is returning `null`. Perhaps the props that you pass to it are incorrect?

Reply:
I've cleared this one as the params has changed, and I did not run it since then